packages/sdk/src/endpoints/IncomingRequestsEndpoint.ts

```diff
@@ -16,7 +17,10 @@ export class IncomingRequestsEndpoint extends Endpoint {
         return await this.put(`/api/v2/Requests/Incoming/${requestId}/CanAccept`, request);
     }
 
-    public async accept(requestId: string, request: AcceptIncomingRequestRequest): Promise<ConnectorResponse<ConnectorRequest>> {
+    public async accept(requestId: string, request: Omit<AcceptIncomingRequestRequestNew, "requestId">): Promise<ConnectorResponse<ConnectorRequest>> {
+        return await this.put(`/api/v2/Requests/Incoming/${requestId}/Accept`, request);
+    }
+    public async acceptold(requestId: string, request: AcceptIncomingRequestRequestOld): Promise<ConnectorResponse<ConnectorRequest>> {
         return await this.put(`/api/v2/Requests/Incoming/${requestId}/Accept`, request);
     }
```

Comment:
runtime is too bloated. just remove the acceptold method and use the new one if it's better IMO

Reply:
with use the new one i mean "update the old one that it gets like the other type"

Reply:
Yea, but the types in the SDK are just wrong. And are not checked against the real accepted types of the runtime. Also, if there is a new subclass in the runtime that comes from time to time this is not noticeable in the SDK.

Reply:
cool, when they are wrong you don't have to provide an "old" and "new" and can just update the wrong types.

Reply:
Yea but the new type is the type of the @nmshd/runtime

Reply:
what's the problem with updating the old type? I don't get that...

Reply:
There is no problem but for, i don't know how long due to the mono repo. There was no way in the SDK to create a
AcceptReadAttributeRequestItemParametersWithExistingAttributeJSON
AcceptProposeAttributeRequestItemParametersWithExistingAttributeJSON
AcceptFreeTextRequestItemParametersJSON

And I think if we really stay with copying (recreating) all the types from runtime in the SDK, this will appear in many other places.

How about we bundle that SDK and import the runtime types so that not all the runtime code is used?

Reply:
It's the plan to have a package only containing types that is used by the runtime and the sdk - but we aren't there yet. for now

1. allow any / or make the type way more open
2. copy the types for now